Sources/NIOCore/ChannelPipeline.swift

```diff
@@ -175,11 +175,12 @@ public final class ChannelPipeline: ChannelInvoker {
     ) -> EventLoopFuture<Void> {
         let future: EventLoopFuture<Void>
 
+        let syncPosition = ChannelPipeline.SynchronousOperations.Position(position)
```

Comment:
This conversion isn't right: we can't create the synchronous position object until we're on the loop.

Reply:
I assume we use the `sending` property here.

Reply:
We do not, and should not. We should be very reluctant to adopt `sending` until we have entirely Swift 6 usage, or it gets very easy for users to accidentally write code that works in one Swift version but not older ones.